.github/renovate.json

```diff
@@ -1,11 +1,16 @@
 {
 	"extends": ["github>balena-io/renovate-config"],
-	"prConcurrentLimit": 1,
+	"prConcurrentLimit": 3,
```

Comment:
Why are you increasing this?

Reply:
B/c I expect that `prHourlyLimit: 1` might help enough and having just 1 starving PR sounds like renovate will always be blocked. I can start w/ leaving `prConcurrentLimit: 1` as it was and only increase it later if you prefer.
Yea let me do that...